settings.yml

```diff
@@ -86,17 +90,17 @@ membership:
 home:
   # update to change the header text
   header: >
-    <h1>OSPO Landscape</h1> The OSPO landscape (<a target="_blank" href="images/landscape.png">png</a>, <a target="_blank"
+    <h1>Magma Landscape</h1> The Magma landscape (<a target="_blank" href="images/landscape.png">png</a>, <a target="_blank"
     href="images/landscape.pdf">pdf</a>) is dynamically generated below. It is modeled after the <a href="http://landscape.cncf.io">CNCF landscape</a> and based
     on the same open source code.
   # don't change this
   footer: |
     Crunchbase data is used under license from Crunchbase to CNCF.
 # ads for the landscape. Good opportunity to advertise your project events
 ads:
-  - url: https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_source=ospo_landscape&utm_medium=web&utm_campaign=kccnceu20
```

Comment:
While it's odd - I'd not revert these changes for now. If you want, you can work with CS to put an add here for a Magma event as it makes sense.